### DISCUSSION TITLE
Can't connect to the peer. No route to host.

### DISCUSSION THREAD

**Discussion by Nkorchi** [tendermint/tendermint#9298] [Q&A]
I have four VMs, each running BigchainDB and MongoDB inside a Docker container and a Tendermint installation in the VM. I am trying to connect the four instances. When running tendermint node, I get several errors that look like this:

E[2022-08-07|22:00:29.410] Dialing failed module=pex addr=4c9fa2f3c6620c1b96ed2ddfb004b977736f8f76@152.7.177.221:26656 err="dial tcp 152.7.177.221:26656: connect: no route to host" attempts=0

When all three of the services (Mongo, Bigchain, Tendermint) are containerized, it does work once I introduce port forwarding, but as-is (which is my specific use case), the nodes cannot seem to find each other. I made sure BigchainDB is listening (inside the container) using lsof -i :26658, and this port is forwarded. I also tried using the persistent_peers setting, but that doesn't seem to do it either.

How do I connect these VMs?

**Comment by thanethomson** [comment]
If it works when you introduce port forwarding, but doesn't when you disable port forwarding, that sounds like a networking issue to me (i.e. not a Tendermint issue).
